.github/workflows/new-pull-request.yaml

```diff
@@ -0,0 +1,30 @@
+name: Run CI for ISV Addons
```

Comment:
Does it require approval from the core team to run? Have you looked into that?

Reply:
Yes. I have that setting in the `Fork pull request workflows from outside collaborators` to ask for approval for all outside contributors. So we are good with that.